libsolidity/interface/StandardCompiler.cpp

```diff
@@ -613,6 +615,8 @@ std::variant<OptimiserSettings, Json::Value> parseOptimizerSettings(Json::Value
 
 }
 
+char const queryKeyDelimiter = '#';
```

Comment:
Hmm this feels a bit off place. I think we can inline where it's used

Reply:
You mean just use '#' character on both places where this is needed and do not introduce the constant?

Reply:
Yea